.dockerignore

```diff
@@ -0,0 +1,6 @@
+venv

```

Comment:
Fix checks that are failing

Reply:
new line?

Reply:
You already fixed it, there was requirements issue